predicators/utils.py

```diff
@@ -905,6 +905,7 @@ def run_policy(
         task_idx: int,
         termination_function: Callable[[State], bool],
         max_num_steps: int,
+        init_state: State = None,
```

Comment:
i'm surprised mypy doesn't complain but I think the type should be `Optional[State]`